apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -22,6 +22,8 @@ import "components"
 Page {
     id: page
 
+    readonly property int _textSwitchGlassCentre: Math.round((permissionIcon.width - Theme.itemSizeExtraSmall) / 2.0)
```

Comment:
maybe _textSwitchIconCentre -- err, I mean _textSwitchIconCenter as I think we are supposed to use US english -- would be a bit clearer, as "glass" may be used as either an adjective or a noun

Reply:
Thanks; I've changed it as you suggested. The naming is awkward because it doesn't really have a practical purpose on its own, it  just happens to be useful for the calculation. I don't want it to give it a name that might lead someone to try to use it. `_textSwitchIconCenter` is suitable vague though ;)